### DISCUSSION TITLE
Typo in REST API error response message

### DISCUSSION THREAD

**Discussion by peterhoogendijk** [community/community#59628] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When creating an organization invitation for a user that is already part of the organization, the error 422 message contains a typo: 

- A user with this email addre**sss** is already a part of this org

I decided to fix this rather unimportant typo myself, as the GitHub code is available on GitHub. But I found that the code in all https://github.com/github repositories does not contain the string `email addresss`. Is the GitHub REST API not kept in GitHub, or is this message composed in another way by the REST API code?

**Comment by KingBain** [comment]
just noticed this to using the GH CLI 

```
{
  "message": "Validation Failed",
  "errors": [
    {
      "resource": "OrganizationInvitation",
      "code": "unprocessable",
      "field": "data",
      "message": "A user with this email **addresss** is already a part of this organization"
    }
  ],
  "documentation_url": "https://docs.github.com/rest/orgs/members#create-an-organization-invitation",
  "status": "422"
}
```

**Comment by thomassiard** [comment]
GitHub hosts the code for specific open-source projects (such as [GitHub CLI](https://github.com/cli/cli) or [hubot](https://github.com/github/hubot)), but core parts of the platform, including the REST API backend, are proprietary.

The typo in the error message might be generated by internal code that is inaccessible to the public. Error messages like this are often dynamically generated in the backend code, and while the public repositories don’t contain this code, GitHub Support can be notified of issues like typos, which can then be addressed by their engineering team.

To report this issue:

Go to the GitHub Support page (in GitHub, scroll to the bottom and click on Contact GitHub Support).
Describe the issue and mention the typo in the 422 error message for organization invitations.
